src/v/cluster/self_test_rpc_types.h

```diff
@@ -255,7 +258,8 @@ struct self_test_result
         fmt::print(
           o,
           "{{p50: {} p90: {} p99: {} p999: {} max: {} rps: {} bps: {} "
-          "timeouts: {} test_id: {} name: {} info: {} type: {} duration: {}ms "
+          "timeouts: {} test_id: {} name: {} info: {} type: {} start_time: {} "
+          "end_time: {} duration: {}ms "
```

Comment:
suggestion(non blocking): could throw a "Z" on the start/end time ints to indicate unix time, though I'm not sure how useful that is in practice.